### PR TITLE
submission endpoint returns creator as well

### PR DESCRIPTION
**Pull request by mozsarmate**

### CODE REVIEW

**Review by FearsomeRover — APPROVED** [review]
lgtm

File: packages/client/.eslintrc.cjs [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = {
   rules: {
     'simple-import-sort/imports': 'error',
     'simple-import-sort/exports': 'error',
-    'prettier/prettier': 'error',
+    'prettier/prettier': 'off',
```
Comment (Tschonti) [inline packages/client/.eslintrc.cjs:16]: do we really want to turn this off? Are the line endings still causing issues?

**Review by Tschonti — COMMENTED** [review]
Is this info shown on the frontend? That was also part of #151 , but it's fine if that will be implemented in another PR.

Also, merge to staging first

File: packages/client/.eslintrc.cjs (hunk 1 above)
Comment (mozsarmate) [reply packages/client/.eslintrc.cjs:16]: yes, unfortunately :(

Comment (Tschonti) [reply packages/client/.eslintrc.cjs:16]: you can try adding the .gitattributes file to the root dir, similar to the newbie projects

**Pull request merged**